### DISCUSSION TITLE
How to add user accounts?

### DISCUSSION THREAD

**Discussion by markseuffert** [datenstrom/community#7] [Ask a question] [answered]
Installation comes without a default user. What's a good way to add one?

**Comment by markseuffert** [comment, marked as answer]
You can add user accounts from the command line:
1. Open a terminal window.
2. Go to your Yellow installation, where the yellow.php is.
3. Type the following line: `php yellow.php user add email@example.com horsebatterystaple`  

Now you can login with `email@example.com` and password `horsebatterystaple`.

See also https://github.com/annaesvensson/yellow-edit

**Comment by ofisia** [comment]
I've tried it, but it didn't work. Am I doing it wrong?
![image](https://cloud.githubusercontent.com/assets/831030/3486180/c49834f6-0424-11e4-83b6-b6a5b9d63d1e.png)

**Comment by markseuffert** [comment]
Jimmy, you can use `cd` to change directory and then `ls` to see files:

```
cd /Applications/MAMP/htdocs/yellowtest
ls
```

**Comment by nibreh** [comment]
HI - And what is the way to add user without a terminal ? I mean by editing user.ini ? THX

**Comment by nibreh** [comment]
Ok, I used a terminal after install php5-cli ... sorry for my ignorance ;)

**Comment by markseuffert** [comment]
You can add a default user by editing the configuration:
1. Go to your Yellow installation, where the yellow.php is.
2. Open file `system/config/user.ini` and add the following line to the end:

`demo@demo.com,$2y$10$8Jow5LJHDPOy6bv5IsHSi.3aF.mZqlSVN6VOVXvl/EsPEba7XVsba,Demo,en,/`

Now you can login with `demo@demo.com` and password `demo`.

**Comment by ofisia** [comment]
As a noob, I find editing the `user.ini` file is far easier than using terminal. Is there an online tool you could recommend to generate that kind of hashed password?

**Comment by markseuffert** [comment]
Here please, an online user account tool: ~~`https://demo.datenstrom.se/tests/new-user`~~ 

**Edit**: Online tool has been removed, you can now create a user account in a web browser.
